gradle/help.gradle

```diff
@@ -30,7 +30,7 @@ configure(rootProject) {
       ["Git", "help/git.txt", "Git assistance and guides."],
       ["ValidateLogCalls", "help/validateLogCalls.txt", "How to use logging calls efficiently."],
       ["IDEs", "help/IDEs.txt", "IDE support."],
-      ["Docker", "help/docker.txt", "Building Solr Docker images."],
```

Comment:
Do you think it makes sense to scatter those help files? I see some benefits of keeping them in one place.

Reply:
I was moving based off of a comment from @dsmiley. I could go either way. I do think it makes sense that the documentation for the docker module resides in the docker module. The other help files in that directory seem to be applicable project-wide, not for a specific module.

Reply:
Ok, fine.